admin/scripts/modules/aktivity/prihlaseni-drd.php

```diff
@@ -20,15 +20,29 @@
     $mail->predmet('Gamecon: umístění družiny v MDrD');
     $druzina = $a->tym()->nazev();
 
+    // E-maily účastníkům
     foreach ($a->prihlaseni() as $uc) {
-        $mail->text("Ahoj " . $uc->nick() . ",\n
+        $mail->text(
+            "Ahoj " . $uc->nick() . ",\n
             bohužel, tvoje družina " . $druzina . " nepostoupila do dalšího kola. Herní bloky, původně 
             rezervované pro turnaj MDrD, jsou nyní volné a můžeš se tak přihlásit na jinou aktivitu.\n
             Díky a s pozdravem,\n 
             tým MDrD");
         $mail->adresat($uc->mail());
         $mail->odeslat();
     }
+
+    // E-mail PJovi
+    foreach (Uzivatel::zIds($a->getOrganizatoriIds()) as $pj) {
```

Comment:
Třída `Aktivita` už v sobě má funkci `\Aktivita::organizatori`, která vrací objekty `Uzivatel` s organizátory. Použil bych tu.

```php
foreach ($a->organizatori() as $pj) {
```

Reply:
Supr; a pochopil jsem teda dobře, že jen PJové jsou tady vedení jako orgové aktivity, jo? Nepřijde to i někomu, koho to "nezajímá"?

Reply:
Mám silný dojem, že tak to je. Ale nejsem si stopro jist. Takže správný dotaz, zkopíruju ho do Trella.